### PR TITLE
[Spanish] Localize Monolitic Applications

### PR DESCRIPTION
**Pull request by mdf-ido**
Signed-off-by: Mo Figueroa <moisesdfigueroa@gmail.com>

### Describe your changes
Translates Monolithic Apps in Spanish

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/monolithic-apps.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+title: Aplicaciones monolíticas
+status: Feedback Appreciated
+category: concepto
+tags: ["arquitectura", "fundamental"]
+---
+
+## ¿Qué es?
+
+Una aplicación monolítica contiene toda la funcionalidad en un único programa desplegable. Suele ser el punto de partida más sencillo y fácil a la hora de crear una aplicación. Sin embargo, una vez que la aplicación crece en complejidad, los monolitos pueden ser difíciles de mantener. Mientras más desarrolladores trabajen en la misma base de código, 
+la probabilidad de conflictos y la necesidad de comunicación interpersonal entre desarrolladores se aumenta.
```
Comment (krol3) [inline content/es/monolithic-apps.md:11]:
```suggestion
la probabilidad de conflictos y la necesidad de comunicación interpersonal entre desarrolladores aumenta. 
```

**Review by krol3 — COMMENTED** [review]
LGTM small changes!

**Review by raelga — APPROVED** [review]

File: content/es/monolithic-apps.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: Aplicaciones monolíticas
+status: Feedback Appreciated
+category: concepto
+tags: ["arquitectura", "fundamental"]
```
Comment (krol3) [inline content/es/monolithic-apps.md:5]: Por convención, los tags los dejamos en ingles

**Review by krol3 — COMMENTED** [review]
Tenemos unas convenciones en el glossario espanol, vamos a crear unas notas para dejar claro esa parte.

File: content/es/monolithic-apps.md (hunk 2 above)
Comment (electrocucaracha) [inline content/es/monolithic-apps.md:5]:
```suggestion
title: Aplicaciones monolíticas
status: Completed
category: Concepto
tags: ["architecture", "fundamental", ""]
```

File: content/es/monolithic-apps.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+## ¿Qué es?
+
+Una aplicación monolítica contiene toda la funcionalidad en un único programa desplegable. Suele ser el punto de partida más sencillo y fácil a la hora de crear una aplicación. Sin embargo, una vez que la aplicación crece en complejidad, los monolitos pueden ser difíciles de mantener. Mientras más desarrolladores trabajen en la misma base de código,
```
Comment (electrocucaracha) [inline content/es/monolithic-apps.md:10]:
```suggestion
Una aplicación monolítica contiene toda la funcionalidad en un único programa desplegable.
Suele ser el punto de partida más sencillo y fácil a la hora de crear una aplicación.
Sin embargo, una vez que la aplicación crece en complejidad, los monolitos pueden ser difíciles de mantener.
Mientras más desarrolladores trabajen en la misma base de código, 
```

File: content/es/monolithic-apps.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+## ¿Qué es?
+
+Una aplicación monolítica contiene toda la funcionalidad en un único programa desplegable. Suele ser el punto de partida más sencillo y fácil a la hora de crear una aplicación. Sin embargo, una vez que la aplicación crece en complejidad, los monolitos pueden ser difíciles de mantener. Mientras más desarrolladores trabajen en la misma base de código, 
+la probabilidad de conflictos y la necesidad de comunicación interpersonal entre desarrolladores aumenta. 
+
+## Problema que aborda
+
+La división de una aplicación en [microservicios](/microservices/) aumenta su sobrecarga operativa ya que hay más cosas que probar, desplegar y mantener en funcionamiento. Al principio del ciclo de vida de un producto, puede ser ventajoso reducir la complejidad y construir una aplicación monolítica hasta que se determine que el producto tiene éxito.
```
Comment (electrocucaracha) [inline content/es/monolithic-apps.md:15]:
```suggestion
La división de una aplicación en [microservicios](/microservices/) aumenta su sobrecarga operativa
ya que hay más cosas que probar, desplegar y mantener en funcionamiento.
Al principio del ciclo de vida de un producto, puede ser ventajoso reducir la complejidad y construir una aplicación monolítica
hasta que se determine que el producto tiene éxito.
```

File: content/es/monolithic-apps.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+## ¿Cómo ayuda?
+
+Un monolito bien diseñado puede defender los principios "lean" al ser la forma más sencilla de poner en marcha una aplicación. Cuando la aplicación monolítica demuestra su valor de negocio y éxito, se puede descomponer en microservicios. Crear una aplicación basada en microservicios antes de que haya demostrado su valor puede suponer un gasto prematuro de esfuerzos de ingeniería. Si la aplicación no aporta valor, ese esfuerzo se desperdicia.
```
Comment (electrocucaracha) [inline content/es/monolithic-apps.md:19]:
```suggestion
Un monolito bien diseñado puede defender los principios "lean" al ser la forma más sencilla de poner en marcha una aplicación.
Cuando la aplicación monolítica demuestra su valor de negocio y éxito, se puede descomponer en microservicios.
Crear una aplicación basada en microservicios antes de que haya demostrado su valor puede suponer un gasto prematuro de esfuerzos de ingeniería.
Si la aplicación no aporta valor, ese esfuerzo se desperdicia.
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]
Algunos cambios en cuanto a etiquetas y saltos de lineas

**Comment by mdf-ido** [comment]
@krol3 y @electrocucaracha  Gracias por los buenos consejos!

**Review by electrocucaracha — APPROVED** [review]

**Pull request merged**